magento/magento2
# Issue 21446 : Remove Commerce specific files from Open Source codebase.

### Description (*)
Removing design files from Open source codebase which relate to Commerce Edition functionality. A separate PR will be created to add theses to Commerce edition.

### Fixed Issues (if relevant)

1. magento/magento2#21446

### Manual testing scenarios (*)

1. Validate that the files have been removed from the codebase.
2. Ensure that the frontend and admin design are unaffected.
3. In Commerce edition, link the codebases with the other PR in effect and ensure that the Commerce specific functionality still appears correctly after deploying static content.

### Contribution checklist (*)
